pkg/sink/sink.go

```diff
@@ -74,26 +74,32 @@ func (r Resource) HandleEvent(response http.ResponseWriter, request *http.Reques
 
 	// Execute each Trigger
 	for _, trigger := range el.Spec.Triggers {
-		go r.executeTrigger(event, request, trigger, eventID)
+		resources, err := r.prepareExecuteTrigger(event, request, trigger, eventID)
+		if err != nil {
+			log.Print(err)
+			response.WriteHeader(http.StatusInternalServerError)
+			return
```

Comment:
I could be misinterpreting this, but I think that there will be an error if we return here. Consider the case where I have 2 triggers in my EventListener, `repo1` and `repo2`, with an interceptor for validation. When the EventListener receives an event from `repo2`, the first trigger's interceptor will fail because the request from `repo1` has a different secret than `repo2`. Then, this code will return with an error code. This behavior is faulty because the `repo2` will never be executed.